axlearn/cloud/gcp/measurement.py

```diff
@@ -37,10 +40,20 @@ class Config(measurement.Recorder.Config):
         Attributes:
             upload_dir: Directory to store metrics for the monitor.
             upload_interval: Time interval (seconds) for monitoring uploads.
+            step_deviation_interval_seconds: Time interval (seconds) for step
+              deviation metrics uploads. -1 to disable step deviation uploads.
```

Comment:
```suggestion
                deviation metrics uploads. -1 to disable step deviation uploads.
```
nit -- we use 4-space tabs.

Reply:
Done!